tools/Localizations/Sources/Localizations/ExportTask.swift

```diff
@@ -67,6 +61,30 @@ struct ExportTask {
         try! task.run()
         task.waitUntilExit()
     }
+    
+    // Get Locales that are in the Xcode project
+    private func getProjectLocales() -> (Array<String>) {
```

Comment:
Could I get some feedback on this code please? it may be very rudientary, it works but learning as I write it